inginious/frontend/plugins/manual_scoring/pages/templates/manual_scoring.html

```diff
@@ -113,19 +113,33 @@ <h4 class="my-1">$:_("Rubric")</h4>
     </table>
         </div>
     <div>
-        <strong>$:_("Current Score: ")<strong style="color: #cc0808" id="output">$data['grade']</strong></strong>
+        <h4><strong>$:_("Current Score: ")<strong id="output" style="color: #002a95">$data['grade']</strong></strong></h4>
```

Comment:
Is this color fixed? Or is it updated when fields in the rubric are selected?

Reply:
It's a default color. The color changes when the rubric is modified. The first time is when the rubric is loaded, but if there is not  information about the rubric, the rubric don't change and keeps that color.

Reply:
ok, no problem.